runtime/compiler/env/CHTable.cpp

```diff
@@ -1015,6 +1131,16 @@ TR_ClassQueries::collectImplementorsCapped(TR_PersistentClassInfo *clazz,
    {
    if (comp->getOption(TR_DisableCHOpts))
       return maxCount+1; // fail the collection
+
+#if defined(J9VM_OPT_JITSERVER)
+   if (comp->isOutOfProcessCompilation())
+      {
+      CollectResolvedImplementors classCollector(comp, clazz->getClassId(), maxCount, callerMethod, slotOrIndex, useGetResolvedInterfaceMethod);
+      classCollector.visit(clazz->getClassId(), locked);
+      classCollector.cacheResolvedMethods();
+      }
+#endif
+
```

Comment:
If we exceeded `maxCount` at this point, we don't have to execute the second pass through the hierarchy.